### ISSUE TITLE
[P1] Coupled scale landing returns a sweep after its own full-cost check detects an increase

### ISSUE THREAD

**Issue by RobTand** [RobTand/tessera#232]
Reviewed ea72d23cbf26a453d0d740afac537fb6d6b8afbc. This affects the opt-in refit_coupled_landing path, not an assertion that current default recipes regress.

_coupled_landing mutates C/I through a sweep using local FP32 gain calculations. At encode.py:2005-2006, a freshly recomputed full quadratic that is greater than or equal to the previous sweep cost causes a break. The function does not restore the preceding accepted plane, and returns the mutated C/I and increased cost at :2035-2037. Its caller takes this result without a comparison at :1919-1923, relying explicitly on mathematical monotonicity. Numerical disagreement between incremental gains and recomputed costs therefore bypasses the guard.

A cheap CPU witness uses the production half width of 16, finite E2M1-compatible unscaled codes, a normal E4M3 table, and an SPD Hessian:

```python
import torch
from tessera.encode import _coupled_landing

W = torch.zeros(1, 32)
W[0, 0], W[0, 16] = 32768.8515625, -32767.51953125
U = torch.zeros(1, 32)
U[0, 0] = U[0, 16] = 1
Ub = U.reshape(1, 2, 16)
H = torch.eye(32)
H[0, 16] = H[16, 0] = 1 - 2**-15
A = torch.ones(1, 2)
table = torch.tensor([.5, .5625, .625, .6875, .75, .8125, .875, .9375,
                      1., 1.125, 1.25, 1.375, 1.5, 1.625, 1.75, 1.875])
I = torch.tensor([[7, 3]])
C = table[I]
E = W - C.repeat_interleave(16, dim=1) * U
before = ((E @ H) * E).sum().item()
new_C, _, after, record = _coupled_landing(
    W, U, Ub, H, A, C, I, table, 16, before)
print(before, after, new_C, record)
# 65536.4140625 -> 65536.6015625
# C [.9375, .6875] -> [1.625, .6875], one sweep / one move
# H minimum eigenvalue = 3.0517578125e-05
```

Run from the checkout with PYTHONPATH=src and native math threads at one. Reproduced on torch 2.10.0+cpu. The returned objective is 24 FP32 ULPs higher than the accepted starting cost. This witness proves failure of the code's own acceptance and monotone-cost contract. It does not prove mathematical or served-quality degradation: recomputing this particular witness in float64 actually lowers the exact quadratic. That distinction should be preserved.

tests/test_ldlq_lut_plane.py:586-627 checks first-order convergence and monotonicity on a few ordinary random fixtures. It does not force disagreement between the local FP32 gain and recomputed full objective, or assert rollback after an increasing sweep.

Fix direction: retain the last accepted C/I, roll back a sweep whose authoritative recomputed cost rises, and compare the final coupled candidate against the prior accepted plane at the caller. Use an explicitly stable objective evaluation and numerical stopping rule for ill-conditioned inputs rather than trusting the local-gain sign as a sufficient acceptance proof. Acceptance should force the increasing-sweep branch and check both returned planes and reported costs; a float64 reference should distinguish a noisy cost from true loss worsening.

No fix was attempted because the user explicitly requested an audit and issues without implementation changes. No GPU/model validation was launched.

Disposition: filed for a separate implementation agent. The user explicitly requested an audit and issues, with no fixes; no implementation changes were made for this finding.
